certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/message/model/Answer.java

```diff
@@ -23,4 +23,21 @@ public class Answer {
   private MessageStatus status;
   private MessageContactInfo contactInfo;
   private Staff authoredStaff;
+
+  public void send(Staff staff, Content content) {
+    this.status = MessageStatus.SENT;
+    this.content = content;
+    this.authoredStaff = staff;
+    this.sent = LocalDateTime.now();
+  }
+
+  public void delete() {
+    this.status = MessageStatus.DELETED_DRAFT;
+  }
+
+  public void save(Staff staff, Content content) {
+    this.content = content;
```

Comment:
Shouldn't modified be updated?

Reply:
Its not done in Webcert and the value is not displayed in frontend, but I think it sounds reasonable to update it. Done!